cmd/istio-cni/main.go

```diff
@@ -37,6 +37,7 @@ var (
 	nsSetupProg   = "istio-iptables.sh"
 
 	injectAnnotationKey = "sidecar.istio.io/inject"
+	injectIstioKey      = "sidecar.istio.io/status"
```

Comment:
Maybe move `sidecarStatusKey` declaration in redirect.go to here and use it instead?